Luis.FuzzyMatching/Solution/FuzzyMatching.Core/RuntimeClient.cs

```diff
@@ -58,7 +58,11 @@ private List<MatchResult> MatchEntitiesWithIndicesPreTokenizeApproach(ProcessedD
             var similarityValuesMatrix = DotProductCalculator.GetDotProduct(inputTokensTFIDFMatrix, processedDataset.TFIDFMatrix, matrix2Abs: processedDataset.TFIDFMatrixAbsoluteValues);
 
             // filter results
-            return MatchFilter.FilterByThresholdBatch(similarityValuesMatrix, dataset, sentenceTokens, threshold);
+            var thresholdMatches = MatchFilter.FilterByThresholdBatch(similarityValuesMatrix, dataset, sentenceTokens, threshold);
+
+            //Edit Distance
+            var finalMatches = MatchFilter.FilterByEditDistance(thresholdMatches, dataset, 0.8f); //still need to decide the threshold, give the user the ability to specify the threshold??
```

Comment:
move this 0.8f threshold value to the configs file
and introduce a variable in the RuntimeClient call it "PostprocesingMatchThreshold" 
similar to this [one](https://github.com/microsoft/CognitiveServicesLanguageUtilities/blob/bae5cf79d7c4383458d2824dc5816b15a43e40dd/Luis.FuzzyMatching/Solution/FuzzyMatching.Core/RuntimeClient.cs#L18)